lxd/devices.go

```diff
@@ -314,6 +314,106 @@ func deviceNetlinkListener() (chan []string, chan []string, chan device.USBEvent
 	return chCPU, chNetwork, chUSB, chUnix, nil
 }
 
+/*
+ * fillFixedInstances fills the `fixedInstances` map with the instances that have been pinned to specific CPUs.
+ * The `fixedInstances` map is a map of CPU IDs to a list of instances that have been pinned to that CPU.
+ * The `targetCpuPool` is a list of CPU IDs that are available for pinning.
+ * The `targetCpuNum` is the number of CPUs that are required for pinning.
+ * The `loadBalancing` flag indicates whether the CPU pinning should be load balanced or not (e.g, NUMA placement when `limits.cpu` is a single number which means
+ * a required number of vCPUs per instance that can be chosen within a CPU pool).
+ */
```

Comment:
Most of the time we use `//` for multi-line comments as well. We have `/* ... */` comments in some older code. Should we just stick to `//` to be consistent?

/cc @tomponline

Reply:
Agreed, lets stick to use `//`